### ISSUE TITLE
Should I call model:clearState() before/after saving/loading a RNN model?

### ISSUE THREAD

**Issue by vikram-gupta** [jcjohnson/torch-rnn#223]
Hi,

I trained a LSTM model and got a decent validation accuracy after few epochs. 

However, I am not very clear with the correct way to store/restore the states of LSTM. What is the correct sequence of steps to follow? 

Should I call `model:clearState()` before saving the model and also after loading the model for evaluation?
